src/common/Configuration.h

```diff
@@ -53,10 +53,10 @@ namespace SDDM {
         );
         // TODO: Not absolutely sure if everything belongs here. Xsessions, VT and probably some more seem universal
         Section(XDisplay,
-            Entry(ServerPath,          QString,     _S("/usr/bin/X"),                           _S("X server path"));
-            Entry(XephyrPath,          QString,     _S("/usr/bin/Xephyr"),                      _S("Xephyr path"));
-            Entry(XauthPath,           QString,     _S("/usr/bin/xauth"),                       _S("Xauth path"));
-            Entry(SessionDir,          QString,     _S("/usr/share/xsessions"),                 _S("Session description directory"));
+            Entry(ServerPath,          QString,     _S(PREFIX "/bin/X"),                           _S("X server path"));
```

Comment:
You are right, yes: a non-default CMAKE_INSTALL_PREFIX would've broken the search for `xauth`, `xsessions`, etc. I've changed the patch to explicitly look for the `share/xsessions` and use that as an X prefix.